protos/notify.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package protos;
+
+option go_package = "github.com/streamdal/snitch-protos/build/go/protos";
+
+enum NotificationType {
+  NOTIFICATION_TYPE_UNSET = 0;
+  NOTIFICATION_TYPE_SLACK = 1;
+  NOTIFICATION_TYPE_EMAIL = 2;
+  NOTIFICATION_TYPE_PAGERDUTY = 3;
+}
+
+message NotificationConfig {
+  optional string id = 1;
+
+  optional string name = 2;
```

Comment:
`id` should be optional because you won't know `id` when you are creating a notification for the first time. But `name` and `type` should not be optional.